packages/schematics/src/collection/application/files/__directory__/package.json

```diff
@@ -35,14 +37,16 @@
     "@angular/platform-browser-dynamic": "<%= angularVersion %>",
     "@angular/router": "<%= angularVersion %>",
     "core-js": "^2.4.1",
+    "graphviz": "0.0.8",
```

Comment:
I'd add them as dependencies to @nrwl/schematics package and remove them from here.

Reply:
can you let me know the path to this config?

Reply:
Sorry, I wasn't clear. I mean this file: `packages/schematics/package.json`